devel/git-crypt/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+name                    git-crypt
+homepage                https://www.agwa.name/projects/git-crypt/
+version                 0.6.0
+github.setup            AGWA ${name} ${version}
```

Comment:
`github.setup` will take care of `name` and `version`, so you should put them here directly.